assets/js/meta-box.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+( function( $ ) {
+
+	$( document ).ready( function() {
+
+		$( 'input[name="primer-layout-override"]' ).change( function() {
+
+			if ( 1 == $( this ).val() ) {
+
+				$( '.primer-layout ul li' )
+					.removeClass( 'disabled' )
+					.addClass( 'active' )
+					.find( 'input' )
+					.prop( 'disabled', false );
+
+			} else {
```

Comment:
😄 fixed in https://github.com/godaddy/wp-primer-theme/commit/911aa339b98a62cc47e560d3605726b1530e0164